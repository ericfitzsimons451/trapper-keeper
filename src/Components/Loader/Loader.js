import React from 'react'
import { NavLink } from 'react-router-dom'
import './Loader.scss'

export const Loader = () => {
	return (
		<div className="loader">
			<h2 className="welcome">Welcome to Trapper Keeper!</h2>
			<p className="description">Trapper Keeper is a wonderful way to take notes and store ideas.</p>
			<h4 className="suggestion">Start by creating a new note now!</h4>
			<NavLink to='/new-note' className='button'>Create New Note</NavLink>
		</div>
	)
}

export default Loader