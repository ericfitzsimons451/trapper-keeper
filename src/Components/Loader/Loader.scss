.loader {
  width: 50vw;
  height: 50vh;
  background-color: #e6d583;
  background-image: 
      url("https://www.transparenttextures.com/patterns/asfalt-dark.png");
  margin: 2rem auto 0;
  border-radius: 15px;
  display: flex;
  flex-direction: column;
}

.welcome {
  padding: 2.5rem .5rem 0 .5rem;
  margin-top: 0; 
}

.description {
  margin: 0 10% 0 10%;
}

.suggestion {
  font-size: 2.5rem;
  margin: 0;
}

.button {
  border: 1px solid;
  border-radius: 15px;
  padding: 20px 0 20px 0;
  margin: 0 10% 0 10%;
  text-decoration: none;
  background-color: #f0b8ec;
  background-image: 
    url("https://www.transparenttextures.com/patterns/dark-tire.png");
}

@media only screen and (max-width: 900px) {
  .suggestion {
    font-size: 1.8rem;
  }
}

@media only screen and (max-width: 480px) {
  .suggestion {
    font-size: 1.3rem;
  }
  .button {
    padding-top: 1rem;
  }
}

