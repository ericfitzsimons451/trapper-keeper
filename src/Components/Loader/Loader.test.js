import { Loader } from './Loader'
import React from 'react'
import { shallow } from 'enzyme'

describe('Loader', () => {
	it('should match the snapshot', () => {
		let wrapper = shallow(<Loader />)
		expect(wrapper).toMatchSnapshot()
	})
})
