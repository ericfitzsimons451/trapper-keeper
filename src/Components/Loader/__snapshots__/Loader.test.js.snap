// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should match the snapshot 1`] = `
<div
  className="loader"
>
  <h2
    className="welcome"
  >
    Welcome to Trapper Keeper!
  </h2>
  <p
    className="description"
  >
    Trapper Keeper is a wonderful way to take notes and store ideas.
  </p>
  <h4
    className="suggestion"
  >
    Start by creating a new note now!
  </h4>
  <NavLink
    className="button"
    to="/new-note"
  >
    Create New Note
  </NavLink>
</div>
`;
